### PR TITLE
fix shape error on cuda

### PR DESCRIPTION
**Pull request by jpchen**
addresses #1381

### CODE REVIEW

File: pyro/contrib/tracking/extended_kalman_filter.py [hunk 1]
```diff
@@ -187,7 +187,7 @@ def update(self, measurement):
         S = H.mm(P).mm(H.transpose(-1, -2)) + R  # innovation cov
 
         K_prefix = self._cov.mm(H.transpose(-1, -2))
-        dx = K_prefix.mm(torch.gesv(dz, S)[0]).squeeze(1)  # K*dz
+        dx = K_prefix.mm(torch.gesv(dz.unsqueeze(1), S)[0]).squeeze(1)  # K*dz
```
Comment (neerajprad) [inline pyro/contrib/tracking/extended_kalman_filter.py]: Just for my understanding - could you comment on why this is failing on the GPU but works fine on CUDA?

File: tests/contrib/tracking/test_ekf.py [hunk 2]
```diff
@@ -11,8 +11,6 @@
 from tests.common import assert_equal, assert_not_equal
```
Comment (neerajprad) [inline tests/contrib/tracking/test_ekf.py]: You'll have to remove the pytest and os imports to make the linter happy.

File: pyro/contrib/tracking/extended_kalman_filter.py (hunk 1 above)
Comment (jpchen) [reply pyro/contrib/tracking/extended_kalman_filter.py]: seems to be a pytorch bug, for some reason on cuda the `gesv` operation will properly handle `(N)` tensors but on cuda they need to be `(N, 1)` tensors.  they both return the same `(N, 1)` output

**Comment by neerajprad** [comment]
Thanks for fixing this, @jpchen!

**Pull request merged**